e2e/core/viewer/viewer-component.e2e.ts

```diff
@@ -121,6 +121,21 @@ xdescribe('Viewer', () => {
         done();
     });
 
+    fit('A11Y', () => {
```

Comment:
has `fit`

Reply:
yep actually I don't want merge this PR now but I want more share how this thing works in order to talk anbout it and discuss the best way to procede on it